libs/domains/organizations/feature/src/lib/list-git-tokens/list-git-tokens.tsx

```diff
@@ -0,0 +1,116 @@
+import { useParams } from 'react-router-dom'
+import {
+  BlockContent,
+  Button,
+  Icon,
+  IconAwesomeEnum,
+  LoaderSpinner,
+  Tooltip,
+  Truncate,
+  useModal,
+  useModalConfirmation,
+} from '@qovery/shared/ui'
+import { dateYearMonthDayHourMinuteSecond, timeAgo } from '@qovery/shared/util-dates'
+import GitTokenCreateEditModal from '../git-token-create-edit-modal/git-token-create-edit-modal'
+import { useDeleteGitToken } from '../hooks/use-delete-git-token/use-delete-git-token'
+import { useGitTokens } from '../hooks/use-git-tokens/use-git-tokens'
+
+export function ListGitTokens() {
+  const { organizationId = '' } = useParams()
+  const { openModal, closeModal } = useModal()
+  const { openModalConfirmation } = useModalConfirmation()
+  const { data: gitTokens = [], isFetched: isFetchedGitTokens } = useGitTokens({ organizationId })
+  const { mutate: deleteToken } = useDeleteGitToken({ organizationId })
+
+  return (
+    <BlockContent title="Git tokens" classNameContent="p-0">
+      {!isFetchedGitTokens ? (
+        <div className="flex justify-center p-5">
+          <LoaderSpinner className="w-5" />
+        </div>
+      ) : gitTokens.length > 0 ? (
+        <ul>
+          {gitTokens?.map((gitToken) => (
+            <li
+              key={gitToken.id}
+              className="flex justify-between items-center px-5 py-4 border-b border-neutral-250 last:border-0"
+            >
+              <div className="flex">
+                <Icon name={gitToken.type} width="20" height="20" />
+                <div className="ml-4">
+                  <h2 className="flex text-xs text-neutral-400 font-medium mb-1">
```

Comment:
Not really a `h2` IMO because this isn't the title of a section, more like stylized version of a list.